### DISCUSSION TITLE
Help Me with Stars

### DISCUSSION THREAD

**Discussion by rifqifarhansyah** [community/community#64708] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Can you guys help me, by giving this repo a star (maybe): [redacted from the moderator team]

I need 16 stars to unlock an achievement badge, :))

**Comment by queenofcorgis** [comment, marked as answer]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like asking for stars on a specific repo. As a result, we'll be closing and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
